### PR TITLE
0.6.1 into develop

### PR DESCRIPTION
**Pull request by rtibbles**
The marathon ends!

### CODE REVIEW

File: install_cexts.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def download_package(path, platform, version, implementation, abi, name):
     return_code = subprocess.call([
         'python', 'kolibripip.pex', 'download', '-q', '-d', path, '--platform', platform,
         '--python-version', version, '--implementation', implementation,
-        '--abi', abi, '-i', index_url, name
+        '--abi', abi, '-i', index_url, '{}=={}'.format(name, pk_version)
```
Comment (rtibbles) [inline install_cexts.py:62]: @lyw07 can you check the merge conflict resolutions in this file to make sure they are right? I erred in favour of 0.6.1 as that seemed to be more recently changed.

Comment (lyw07) [reply install_cexts.py:62]: Yes the second line from 0.6.1 is the correct one! It is to fix the version issue for installing cryptography.

File: .buildkite/build_windows_installer.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ make writeversion
 cd $KOLIBRI_DOCKER_PATH
 git clone https://github.com/learningequality/kolibri-installer-windows.git
 cd kolibri-installer-windows
-git checkout develop
+git checkout v1.0.0
```
Comment (indirectlylit) [inline .buildkite/build_windows_installer.sh]: let's switch this back to `develop` and then we're good to go

**Comment by rtibbles** [comment]
Reverted the change, and then pulled in latest 0.6.1. Merge conflict resolution seemed straight forward. Will just check linting is still all fine.

**Pull request merged**